src/hooks/use-add-site.ts

```diff
@@ -110,20 +114,23 @@ export function useAddSite() {
 			setDoesPathContainWordPress,
 			usedSiteNames,
 			loadingSites,
+			fileForImport,
+			setFileForImport,
 		} ),
 		[
-			__,
-			doesPathContainWordPress,
-			error,
 			handleAddSiteClick,
 			handlePathSelectorClick,
-			siteWithPathAlreadyExists,
 			handleSiteNameChange,
-			siteName,
+			siteWithPathAlreadyExists,
 			sitePath,
 			proposedSitePath,
+			__,
+			error,
+			siteName,
+			doesPathContainWordPress,
```

Comment:
We could let these dependencies as in the previous order to avoid introducing changes not needed.

Reply:
Resolved in https://github.com/Automattic/studio/pull/378/commits/f03498c76e9f5878530e72a72ae02a29dc4d21d1